chain/consensus/common/mine.go

```diff
@@ -66,12 +66,13 @@ func PrepareBlockForSignature(ctx context.Context, sm *stmgr.StateManager, bt *l
 	}
 
 	for _, msg := range bt.CrossMessages {
```

Comment:
Why block template does still have CrossMessages?

Reply:
We are just keeping it in the block template for miners to be able to discern and prepare the cross-messages to include in the block (in case that's needed), but this doesn't affect the backward compatibility of the protocol.